tests/overrides/test_data_parallel.py

```diff
@@ -57,18 +57,25 @@ def test_lightning_wrapper_module_warn_none_output(wrapper_class):
     pl_module.training_step.return_value = None
     pl_module.validation_step.return_value = None
     pl_module.test_step.return_value = None
+    pl_module.predict.return_value = None
 
     with pytest.warns(UserWarning, match="Your training_step returned None"):
         pl_module.running_stage = RunningStage.TRAINING
         wrapped_module()
 
-    with pytest.warns(UserWarning, match="Your test_step returned None"):
+    with pytest.warns(None, match="Your test_step returned None") as record:
```

Comment:
is this kind of reverse check that the warning is not raised?

Reply:
`assert not record` is the actual the reverse check

Reply:
You could use something like this https://github.com/kaushikb11/pytorch-lightning/blob/hooks%2Foutputs/tests/deprecated_api/__init__.py#L29

Reply:
will do. thanks!